### PR TITLE
Use PSR4 autoloading for tests.

### PR DESCRIPTION
**Pull request by mvdnbrk**

### CODE REVIEW

File: tests/CacheTest.php [hunk 1]
```diff
@@ -134,6 +138,9 @@ public function testCachesRootToSpecialFilename()
 
         $this->cache->setCachePath('page-cache');
         $this->cache->cache(Request::create('/', 'GET'), Response::create('content'));
+
+        $this->assertEquals('page-cache', $this->cache->getCachePath());
+        $this->assertEquals('page-cache/baz', $this->cache->getCachePath('baz'));
```
Comment (JosephSilber) [inline tests/CacheTest.php]: What are these, and why are they here?

Comment (mvdnbrk) [reply tests/CacheTest.php]:
```
There was 1 risky test:

1) Silber\PageCache\Tests\CacheTest::testCachesRootToSpecialFilename
This test did not perform any assertions
```

**Comment by mvdnbrk** [comment]
Reverted bumping depencencies / tests.

File: tests/CacheTest.php [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 use Mockery as m;
 use Silber\PageCache\Cache;
+use PHPUnit\Framework\TestCase;
 use Illuminate\Container\Container;
 use Illuminate\Filesystem\Filesystem;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
-class CacheTest extends PHPUnit_Framework_TestCase
+class CacheTest extends TestCase
```
Comment (JosephSilber) [inline tests/CacheTest.php]: Is `TestCase` available in our version of PHPUnit?

Comment (mvdnbrk) [reply tests/CacheTest.php]: Yes it is.

`~/vendor/phpunit/phpunit/src/ForwardCompatibility/TestCase.php`

**Pull request merged**